actors/src/main/scala/zio/actors/ActorSystem.scala

```diff
@@ -272,8 +285,23 @@ private[actors] object ActorSystemUtils {
   def buildPath(actorSystemName: String, actorPath: String, remoteConfig: Option[RemoteConfig]): String =
     s"zio://$actorSystemName@${remoteConfig.map(c => c.addr.value + ":" + c.port.value).getOrElse("0.0.0.0:0000")}$actorPath"
 
-  def retrieveConfig(sysName: String, configFile: Option[File]): Task[Option[RemoteConfig]] =
-    configFile.fold[Task[Option[RemoteConfig]]](IO.none)(file => Utils.getConfiguration(sysName, file))
+  def retrieveConfig(configFile: Option[File]): Task[Option[String]] =
+    configFile.fold[Task[Option[String]]](Task.none) { file =>
+      Managed.make(IO(Source.fromFile(file)))(f => UIO(f.close())).use { a =>
```

Comment:
It can be simplified as:

```scala
IO(Source.fromFile(file)).toManaged(f => UIO(f.close()))
```
Keep in mind that it might make sense to run this on blocking pool.